dist/debian/readme

```diff
@@ -1 +0,0 @@
-Put build_deb.sh here.
```

Comment:
can we have a bit more examples here ? on how to build a DEB ? or at least in the main readme

Reply:
Updated README.md.